### ISSUE TITLE
Learn to Code From Home: The Coronavirus Quarantine Developer Skill Handbook

### ISSUE THREAD

**Issue by artur-de-andrade** [artur-de-andrade/Articles#2]
## TL;DR
A Programming Road Map. Free online resources for various skills:
* HTML, CSS and JavaScript
* Computer Science concepts, Algorithms, Data Structures, and Databases
* Python and basic Data Science libraries like NumPy
* Command line tools like Linux, Git, and Bash
* Small business and job application basics

### Article Link
[https://www.freecodecamp.org/news/coronavirus-academy/](https://www.freecodecamp.org/news/coronavirus-academy/)

### Author
Quincy Larson

## Key Takeaways
* It's really up to the needs of the reader. If you want to learn Python, go to the corresponding section and check it out. Same goes for the other skills I listed above.
* Also, if you don't know where to start, try reading the intro to each section. It's got a summary of what you can achieve with that skill.

## Useful Code Snippets
```javascript
/* No code here */
```

## Useful Tools
* Again, it's up to the needs of the reader. Each section has amazing and free online learning resources.

## Comments/ Questions
* The article is an awesome reference to a lot of skills I'm currently pursuing, or at least plan to some day.
